cmd/teleirc.go

```diff
@@ -18,42 +17,43 @@ const (
 
 var (
 	flagPath    = flag.String("conf", ".env", "config file")
-	flagDebug   = flag.Bool("debug", false, "enable debugging")
+	flagDebug   = flag.Bool("debug", false, "disable debugging")
 	flagVersion = flag.Bool("version", false, "displays current version of TeleIRC")
 )
 
 func main() {
 	flag.Parse()
 
+	verbose := internal.Debug { DebugLevel:  *flagDebug }
```

Comment:
I wonder if it's better to just call this `log` or `debug` instead of verbose, so it's explicitly clear this is part of the logging.
I can go either way with this, just a thought.